content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md

```diff
@@ -356,6 +356,156 @@ spec:
 
 {{< /language-toggle >}}
 
+## Backend entities
+
+When a backend [starts up][41], Sensu automatically creates the [`sensu-system` namespace][44] and a new entity with `entity_class: backend`.
+Sensu populates the backend entity with the backend's system information.
+
+The backend uses its own entity to report cluster state errors.
+Backend entities can generate events due to error conditions like unavailable secrets providers or secrets.
+Events generated by a sensu-agent running on the backend host are also associated with the backend entity.
+
+To prevent overloading the event bus with backend events, Sensu generates backend events no more than every 30 seconds, unless the status changes.
+
```

Comment:
I also saw this in the updated code:

```
BackendCheckName is the name fo the check that is created when a backend
	// event is generated from an error.
	BackendCheckName = "backend"
```

Do we need to document this check?

Reply:
Good catch Hilary! This is actually not used anymore and will need to be removed. The idea was that events created against the backend entity would use "backend" as check name, however we ended up going more granular and use something more generic. Currently events are created for Vault access errors and we use "secrets" as the check name.